docs/mocksupport.rst

```diff
@@ -340,17 +340,99 @@ Server InvokeMethod operation method
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 The fake InvokeMethod behaves like :meth:`~pywbem.WBEMConnection.InvokeMethod`.
-Since this method is outside the normal operations in that it generally implies
-some sort of side effect it could not be implemented simply to get objects
-from the repository or put them into the repository.
 
-the fake InvokeMethod requires a callback to a user defined function
-based on the namespace, class, and method defined in the call. Because the
-very nature of invoke method, there was no way to define a standard means
+Since this method is outside the normal intrinsic WBEM operations in that it
+generally implies a side effect and not just a get/put interface to a
+repository.  It could not be implemented simply to get objects from the
+repository or put them into the repository.
+
+The fake InvokeMethod requires a callback to a user defined function
+based on the namespace, objectname, and method defined in the call.\
+Because the nature of InvokeMethod, there was no way to define a standard means
 to get information just from the mock repository for the responses.
 
-TODO define further.
+This user defined function acts as the WBEM server implementation of the
+InvokeMethod, processing input parameters and returning a return value and
+output parameters.
+
+The user must create a function that will be executed as a callback for each
+method that is to be tested. This method is attached to the mock repository
+through :meth:`~pywbem_mock.FakedWBEMConnection.subscribe_method` which
+defines the namespace, class, and methodname for the InvokeMethod and the
+callback method to be executed.
+
+
+This user defined callback method should have the signature defined in
+:meth:`~pywbem_mock.FakedWBEMConnection.subscribe_method`.
+
+Generally callback input and output are as follows:
```

Comment:
I don't think we need to have a brief form of the callback interface at this point. The pointer to the description should be sufficient here.

Reply:
Done